src/components/Root.js

```diff
@@ -37,58 +38,66 @@ const d2Config = {
     ],
 }
 
-const Root = ({ store }) => (
-    <DataProvider
-        config={{
-            baseUrl: process.env.DHIS2_BASE_URL,
-            apiVersion,
-        }}
-    >
-        <ReduxProvider store={store}>
-            <D2Shim d2Config={d2Config} i18nRoot="./i18n_old">
-                {({ d2, d2Error }) => {
-                    if (!d2 && !d2Error) {
+const Root = ({ store }) => {
```

Comment:
Use 
const Root = ({ store }) => ( <DataProvider... )
instead of
const Root = ({ store }) => { return (<DataProvider...

Reply:
Fixed!